components/brave_wallet/common/config.gni

```diff
@@ -1,3 +1,4 @@
 declare_args() {
   brave_infura_project_id = ""
+  zero_ex_api_key = ""
```

Comment:
Many of Brave build flags have `brave_` prefix. Most likely makes sense to add prefix here too.

Reply:
Fixed in https://github.com/brave/brave-core/pull/18033.